### PR TITLE
feat: add jwt check to aggregated

### PR DESCRIPTION
**Pull request by Blightwidow**
Depdends on https://github.com/ukraine-taskforce/infrastructure/pull/19 & https://github.com/ukraine-taskforce/requests-dashboard/pull/49

This will allow checking JWT token and prevent querying the endpoint if not correctly identified

### CODE REVIEW

**Comment by KonstantinTsykulenko** [comment]
I don't think this is going to work with current CI/CD as it just packages contents of each respective lambda folder into a zip file

**Comment by KonstantinTsykulenko** [comment]
> I don't think this is going to work with current CI/CD as it just packages contents of each respective lambda folder into a zip file

Although, maybe it would since you committed node-modules...

**Review by KonstantinTsykulenko — APPROVED** [review]
Please test your branch on dev (available through manually running the GH action) to make sure it actually works, I have some doubts about packaging

File: lambda/requests-aggregated/requests-aggregated.js [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+  userPoolId: process.env.poolId,
+  tokenUse: "access",
+  clientId: process.env.clientId,
+});
+
 module.exports.handler = async (event) => {
-  const AWS = require('aws-sdk')
-  const s3 = new AWS.S3({apiVersion: '2006-03-01'});
+  try {
+    await verifier.verify(event.headers["Authorization"]);
```
Comment (KonstantinTsykulenko) [inline lambda/requests-aggregated/requests-aggregated.js:13]: Shouldn't we need the secret key to verify the token?

Comment (Blightwidow) [reply lambda/requests-aggregated/requests-aggregated.js:13]: They are in the pool that it connects to by the instance of the verifier above

**Pull request merged**